ktor-server/ktor-server-plugins/ktor-server-webjars/jvm/test/io/ktor/server/webjars/WebjarsTest.kt

```diff
@@ -130,6 +135,24 @@ class WebjarsTest {
         }
     }
 
+    @Test
+    fun classifiedAsStatic() {
+        var isStatic = false
+        var location = ""
+        withTestApplication {
```

Comment:
`withTestApplication { ... }` is deprecated and we try to use new API `testApplication { ... }`

Reply:
Oh right I meant to ask - in this case where the whole file is the old way would it be better to retrofit to the new method or just leave what's there?